agent/strategy/quality/techdebt/__init__.py

```diff
@@ -0,0 +1,18 @@
+from agent.strategy.quality.techdebt.lizard_parser import lizardParser
+from agent.strategy.quality.techdebt.cloc_parser import ClocParser
+
+class TechDebt:
+    """
+    runs a variety of techdebt tools on the given repository
+    """
+
+    techdebt_parsers = [
+                        lizardParser(),
+                        ClocParser()
+                    ]
+
+    def run(self, directories):
+        techdebt_dir = directories.mkdir("techdebt")
+
+        for parser in self.techdebt_parsers:
+            parser.run(directories, techdebt_dir=techdebt_dir)
```

Comment:
Please check the updated code to run tools for each file in the repo here: https://github.com/Semalab/agent/blob/SQ-5011-take2/agent/strategy/quality/linters/__init__.py -- this is needed for lizard.

Reply:
Hi @ndabas , this is separately handled in lizard parser. SQ-5115.